packages/microsoft-edge-dev-deb/microsoft-edge-dev-deb.pacscript

```diff
@@ -1,7 +1,8 @@
 name="microsoft-edge-dev-deb"
 gives="microsoft-edge-dev"
-version="99.0.1150.2-1"
+breaks="microsoft-edge-stable-deb microsoft-edge-beta-deb"
+version="100.0.1181.0-1"
 url="https://packages.microsoft.com/repos/edge/pool/main/m/${gives}/${gives}_${version}_amd64.deb"
 description="Want to see what we were working on last week? Our Dev channel offers the latest builds available for Linux"
```

Comment:
Change the description to something else.

Reply:
3eb1396dd215a47d4b4adac8211c54d9e971163e